.github/workflows/publish-to-test-pypi.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Python 🐍 distributions 📦 to TestPyPI
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
@timthelion So this trigger is on pushes to master.

So in theory, a merged PR triggers this workflow.

What this workflow does is build the app and publish it to test.pypi.org

From there you could install it, or just accept that success means that the package is still valid with the new changes to be pushed to pypi once a release is ready.